extension/js/common/settings.ts

```diff
@@ -305,10 +304,7 @@ export class Settings {
           window.location.href = Url.create('/chrome/settings/setup.htm', { acctEmail: response.acctEmail, idToken: response.id_token });
         }
       } else if (response.result === 'Denied' || response.result === 'Closed') {
-        const authDeniedHtml = await Api.ajax({ url: '/chrome/settings/modules/auth_denied.htm' }, Catch.stackTrace()) as string; // tslint:disable-line:no-direct-ajax
-        if (await Ui.modal.confirm(authDeniedHtml, true)) {
-          await Settings.newGoogleAcctAuthPromptThenAlertOrForward(settingsTabId, acctEmail, scopes);
-        }
+        await this.renderSubPage(acctEmail, settingsTabId!, '/chrome/settings/modules/auth_denied.htm', undefined, 450);
```

Comment:
You could simplify this further. `auth_denied` could be reverted back to a HTML snippet instead of a HTML page. You could remove the last sentence that is prompting user to write support could be appended here in this code, and then you can render it as a modal. So all of this code will be:

```ts
const authDeniedHtml = await Api.ajax({ url: '/chrome/settings/modules/auth_denied.htm' }, Catch.stackTrace()) as string; // tslint:disable-line:no-direct-ajax
await Ui.modal.info(`${authDeniedHtml}\n<div class="line">${Lang.general.contactIfNeedAssistance(), true)}`)
```

The current one uses more code and is a bit broken looking in comparison:

![image](https://user-images.githubusercontent.com/6306961/190351032-435420fe-e09e-419f-8d0e-a3342d1842f4.png)